tests/integration/e2e/spicy_swan_performance_test.py

```diff
@@ -211,7 +211,6 @@ def test_benchmark_datasets() -> None:
 
         start_time = time.time()
         publish_ptr = sum_ptr.publish(sigma=500_000)
-        publish_ptr.block_with_timeout(timeout)
```

Comment:
For all of these commented out "block_with_timeout" methods, we should move the timeout parameter to the .get() method instead. Infinite blocking could cause all sorts of issues later on. Additionally, if something does timeout we would do well to make a really obvious stdout so that we can see why CI would be running so slowly.

Reply:
Fixed by: [a8c5abf](https://github.com/OpenMined/PySyft/pull/6918/commits/a8c5abf1494356f854a81631b814e5928bc0eb8b)